app/bundles/CoreBundle/Assets/js/1a.content.js

```diff
@@ -1442,6 +1471,49 @@ Mautic.activateLiveSearch = function (el, searchStrVar, liveCacheVar) {
             mQuery(btn + ' i').removeClass('ri-eraser-line').addClass('ri-search-line');
         }
     }
+
+    /**
+     * Keyboard navigation for Global Search modal.
+     * Supports: ArrowUp, ArrowDown, Tab, Enter, Escape
+     */
+    if (mQuery(el).attr('id') === 'globalSearchInput') {
+        mQuery(el).off('keydown.globalSearchNav').on('keydown.globalSearchNav', function (e) {
```

Comment:
```suggestion
        mQuery(el).off('keydown.globalSearchNav');

        mQuery(el).on('keydown.globalSearchNav', function (e) {
```
nitpick: I’d suggest using this form because it’s more readable. A new line is added to define the handler for the `keydown.globalSearchNav` event, making it easier to comprehend the code.